samples/bluetooth/hci_rpmsg/nrf5340_cpunet_iso-bt_ll_sw_split.conf

```diff
@@ -109,6 +109,8 @@ CONFIG_BT_CTLR_ISO_TX_BUFFERS=8
 CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE=251
 CONFIG_BT_CTLR_ISOAL_SOURCES=2
 
+CONFIG_BT_CTLR_ADVANCED_FEATURES=y
+CONFIG_BT_CTLR_ISOAL_PSN_IGNORE=y
```

Comment:
I think maybe these are not needed after the fix was done for multiple broadcast source streams (in host)